lib/galaxy/tools/data_manager/manager.py

```diff
@@ -37,11 +37,19 @@ def __init__(self, app, xml_filename=None):
                 continue
             self.load_from_xml(filename)
         if self.app.config.shed_data_manager_config_file:
-            self.load_from_xml(self.app.config.shed_data_manager_config_file, store_tool_path=True)
+            try:
+                self.load_from_xml(self.app.config.shed_data_manager_config_file, store_tool_path=True)
+            except (OSError, IOError) as exc:
+                if exc.errno != errno.ENOENT or self.app.config.shed_data_manager_config_file_set:
```

Comment:
@natefoo @ic4f I don't think `self.app.config.shed_data_manager_config_file_set` and `self.app.config.data_manager_config_file_set` are set anywhere.

Reply:
IIRC this is dynamic, but I'll double check.

Reply:
Yeah, they are set by `_parse_config_file_options()`

Reply:
I think they might get set here: 
https://github.com/galaxyproject/galaxy/blob/dev/lib/galaxy/config/__init__.py#L162
Until this: https://github.com/galaxyproject/planemo/pull/950 was merged, that's where  `data_manager_config_file_set` was set (was passed with kwargs as empty string). 
That said, for future consideration, I think that block has too many special cases, which makes it not intuitive. Even a comment explaining the 'why' would help, I think.